schema/deploy/shipping/views.sql

```diff
@@ -3697,44 +3699,57 @@ grant select
-  , encounters.enrollment_date_time as study_enrollment_date_time
-  , encounters.campus_location
-  , encounters.affiliation
-  , encounters.athlete as is_student_athlete
-  , encounters.student_level
-  , encounters.sea_employee_type as employee_category
-  , encounters.inperson_classes as is_taking_inperson_classes
-  , encounters.uw_job as works_at_uw
-  , encounters.on_campus_freq as on_campus_frequency_code
-  , case when encounters.on_campus_freq = 'one_or_less' then 'One day a week or less'
-    when encounters.on_campus_freq = 'two_or_more' then
-    'Two days a week or more' when encounters.on_campus_freq = 'not_on_campus' then
-    'Do not come to campus' else 'Unknown' end as on_campus_frequency_description
-  , encounters.wfh_base as able_to_work_or_study_from_home_code
-  , case when encounters.wfh_base = 'onsite' then 'No, I always have to be on-site for work or school'
-    when encounters.wfh_base = 'only_wfh' then 'Yes, I have only worked or studied from home'
-    when encounters.wfh_base = 'wfh_onsite' then 'I have worked or studied both from home AND on-site'
-    when encounters.wfh_base = 'dont_say' then 'Prefer not to say' else
-    'Unknown' end as able_to_work_or_study_from_home_description
-  , encounters.core_housing_type as housing_type
-  , encounters.core_house_members as number_house_members
-  , encounters.live_other_uw as lives_with_uw_students_or_employees
-  , encounters.uw_apt_yesno as lives_in_uw_apartment
-  , encounters.alerts_off
-
-  from shipping.uw_reopening_encounters_v1 encounters
-  join warehouse.identifier ids on cast(ids.uuid as text) = encounters.collection_identifier
-  join shipping.return_results_v3 results on results.qrcode = ids.barcode
-  where results.status_code in (values ('positive'), ('negative'), ('inconclusive'), ('never-tested'))
+  , age as age_at_encounter
+  , sex
+  , pronouns as preferred_pronouns
+  , text_or_email as preferred_contact_method_for_study
+  , text_or_email_attestation as preferred_contact_method_for_attestations
+  , enrollment_date_time as study_enrollment_date_time
+  , campus_location
+  , affiliation
+  , athlete as is_student_athlete
+  , student_level
+  , sea_employee_type as employee_category
+  , inperson_classes as is_taking_inperson_classes
+  , uw_job as works_at_uw
+  , on_campus_freq as on_campus_frequency_code
+  , case
+      when on_campus_freq = 'one_or_less' then 'One day a week or less'
+      when on_campus_freq = 'two_or_more' then 'Two days a week or more'
+      when on_campus_freq = 'not_on_campus' then 'Do not come to campus'
+      else 'Unknown'
+  end as on_campus_frequency_description
+  , wfh_base as able_to_work_or_study_from_home_code
+  , case
+      when wfh_base = 'onsite' then 'No, I always have to be on-site for work or school'
+      when wfh_base = 'only_wfh' then 'Yes, I have only worked or studied from home'
+      when wfh_base = 'wfh_onsite' then 'I have worked or studied both from home and on-site'
```

Comment:
Is there a reason to change the capitalization there? Otherwise, I would prefer to leave this capitalization consistent

Reply:
Good catch! I Inadvertently changed this when I was lowercasing instances of "AND" that I had added.